pom.xml

```diff
@@ -108,6 +108,7 @@
         <module>extensions-contrib/parquet-extensions</module>
         <module>extensions-contrib/statsd-emitter</module>
         <module>extensions-contrib/orc-extensions</module>
+        <module>extensions-contrib/scan-query</module>
```

Comment:
nit. I heard it's supposed to be sorted. `orc-extentions` was already not like that but we can fix altogether here.